packages/node_modules/@webex/plugin-messages/src/messages.js

```diff
@@ -241,29 +241,6 @@ const Messages = SparkPlugin.extend({
       .then((res) => res.body.items || res.body);
   },
 
-  /**
-   * Marks a message as read.
```

Comment:
We don't want to be able to mark a message as read also? This is how the conversation API works.